ui/src/components/common/PaginationControls.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Pagination } from 'semantic-ui-react'
+
+const PaginationControls = (props: {
+  totalPages: number,
+  onPageChange: (activePage: number) => void
+}) => (
+  <Pagination
+    className='paginataion-controls'
+    boundaryRange={0}
+    defaultActivePage={1}
+    ellipsisItem={null}
+    firstItem={null}
+    lastItem={null}
+    onPageChange={(_, data) => data.activePage && props.onPageChange(Number(data.activePage))}
```

Comment:
fyi there's a neat 'lil shortcut where you can do `+data.activePage` to convert a numerical string to a real number, instead of `Number(data.activePage)`!